option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message LegionApplyRequest
{
 required U64Type legion_id = 1;
}
